Networking/Networking/CookieNonce/CookieNonceAuthenticator.swift

```diff
@@ -155,7 +155,7 @@ private extension CookieNonceAuthenticator {
     }
 
     func buildNonceRequestURL(base: URL) -> URL? {
-        URL(string: "admin-ajax.php?action=rest-nonce", relativeTo: base)
+        URL(string: base.absoluteString + "/admin-ajax.php?action=rest-nonce")
```

Comment:
This can cause failure if the base URL doesn't have a "/" at the end. For example, if the base is `https://example.com/wp-admin`, the result would be `https://example.com/admin-ajax.php?action=rest-nonce`, without `wp-admin`.

I removed the potential backlash at the end of the base URL to make sure we don't have duplicated backlashes in the URL. The change was added in b810c6a1e12ee5b5306e7a09fb2fc012aa661215.